rag_experiment_accelerator/evaluation/eval.py

```diff
@@ -362,6 +390,62 @@ def llm_context_recall(
     ) * 100
 
 
+def setup_ragas(metrics, environment: Environment, config: Config):
+    """
+    Initialises the llm model and embedding model instances to be used for evaluation
+
+    Args:
+        metrics (list): The list of metrics to use for evaluation.
+        environment (Environment): Initialised Environment class containing environment configuration.
+        config (Config): The configuration of the experiment.
+
+    Returns:
+        None
+    """
+
+    azure_configs = {
+        "base_url": environment.openai_endpoint,
+        "model_deployment": config.AZURE_OAI_EVAL_DEPLOYMENT_NAME,
+        "model_name": config.AZURE_OAI_EVAL_MODEL_NAME,
+        "embedding_deployment": config.AZURE_OAI_EVAL_EMBEDDING_DEPLOYMENT_NAME,
+        "embedding_name": config.AZURE_OAI_EVAL_EMBEDDING_MODEL_NAME,
+    }
+
+    azure_model = AzureChatOpenAI(
+        openai_api_version=environment.openai_api_version,
+        azure_endpoint=azure_configs["base_url"],
+        azure_deployment=azure_configs["model_deployment"],
+        model=azure_configs["model_name"],
+        validate_base_url=False,
+    )
+    ragas_azure_model = LangchainLLMWrapper(azure_model)
+
+    azure_embeddings = AzureOpenAIEmbeddings(
+        openai_api_version=environment.openai_api_version,
+        azure_endpoint=azure_configs["base_url"],
+        azure_deployment=azure_configs["embedding_deployment"],
+        model=azure_configs["embedding_name"],
+    )
+    ragas_azure_embeddings = LangchainEmbeddingsWrapper(azure_embeddings)
```

Comment:
No. Since RAGAS can be used with different LLM models these wrappers help the RAGAS library implement a common interface used by the metrics. We could of course create our own implementation of the wrappers, but it would require duplicating a lot of existing logic and continuous monitoring of the RAGAS library for compatibility issues. Is there a specific reason why we would want not to import them?

Reply:
This needs to be evaluated because of new security guidance

Reply:
@ritesh-modi , we have many LangChain dependencies already which we need to resolve with priority,
I think we should open a new separate issue to address this across and not block this PR to prerelease branch.
WDYT?